Makefile

```diff
@@ -68,8 +92,17 @@ code-quality: ## Run linter & formatter.
 
 lint-python-code: ## Run linter for python codes
 	$(info Run linter for python codes)
+	make lint-python-code-quality
+	make lint-python-code-security
+
+lint-python-code-quality: ## Run linter for python codes quality
+	@$(call puts,INFO,"Sniffs code smells in Python code")
 	pylint **/*.py
 
+lint-python-code-security: ## Run linter for python codes security
+	@$(call puts,INFO,"Find common security issues in Python code")
+	bandit **/*.py -s B605,B607,B104
```

Comment:
@miaojun0 must add comment for what B605,B607,B104 is

Reply:
B605 is "start_process_with_a_shell",B607 is "start_process_with_partial_path", both of them are the way to execute system command。B104 is "hardcoded_bind_all_interfaces", just like 0.0.0.0

Reply:
OK. we need to comment these info in the codes so that we can get noticed in future